src/vs/workbench/contrib/chat/browser/chatVariables.ts

```diff
@@ -69,6 +69,10 @@ export class ChatVariablesService implements IChatVariablesService {
 		const resolvedAttachedContext: IChatRequestVariableEntry[] = [];
 		attachedContextVariables
 			?.forEach((attachment, i) => {
+				if (attachment.value instanceof URI && attachment.value.scheme === 'ccreq') {
```

Comment:
This scheme is extension-contributed, can we avoid it living here?

Reply:
Maybe there's another way I can do this besides checking the scheme